lib/wallet/cardano-wallet.cabal

```
@@ -363,6 +363,7 @@ library
     Cardano.Wallet.Version
     Cardano.Wallet.Version.TH
     Cardano.Wallet.Write.Tx
+    Cardano.Wallet.Write.Tx.Built
```

```suggestion
    Cardano.Wallet.Transaction.Built
```

See comment at `module Cardano.Wallet.Write.Tx.Built`.